src/ledger/LedgerManagerImpl.h

```diff
@@ -67,11 +71,13 @@ class LedgerManagerImpl : public LedgerManager
                          CatchupConfiguration::Mode catchupMode);
 
     void processFeesSeqNums(std::vector<TransactionFramePtr>& txs,
-                            AbstractLedgerTxn& ltxOuter, int64_t baseFee);
+                            AbstractLedgerTxn& ltxOuter, int64_t baseFee,
+                            LedgerCloseMeta* ledgerCloseMeta);
```

Comment:
Why take the raw-pointer as argument instead of `std::unique_ptr<LedgerCloseMeta> const&`? Passing the smart-pointer makes ownership questions (eg. do I have to delete this?) much clearer.

Reply:
Ah, good point. Fixed.